app/store/env/go.js

```diff
@@ -1166,6 +1166,29 @@ YUI.add('juju-env-go', function(Y) {
       userCallback(transformedData);
     },
 
+    setCharm: function(serviceName, charmUrl, force, callback) {
```

Comment:
Yes this is pure spike code I wasn't sure where to put it. React views can be thought of as components and because they can be destroyed and modified just as DOM elements they aren't supposed to do anything other than 'call out' to get things done. So this was just a place to put the 'controller' bits of the change-version view.